### ISSUE TITLE
Pushing multiple directories sequentially outputs duplicate log messages

### ISSUE THREAD

**Issue by petter-eikeland** [forcedotcom/cli#1878]
### Summary
When pushing multiple directories with ` "pushPackageDirectoriesSequentially": true`, sfdx outputs duplicate log messages. This clutters the terminal when pushing multiple package (>6)

### Steps To Reproduce:
**Repository to reproduce:** [dreamhouse-lwc](https://github.com/petter-eikeland/dreamhouse-lwc)

1.  Create 2 or more directories, each with e.g. a custom field on the Campaign object
1.  Create a scratch org
1.  Set  `"pushPackageDirectoriesSequentially": true` in sfdx-project.json
1.  Push local changes to scratch org

### Expected result
The directories are pushed sequentially with a single log message: 

Pushing v55.0 metadata to test-ovsqkpand1dp@example.com using the v56.0 SOAP API
DEPLOY PROGRESS | ████████████████████████████████████████ | 2/2 Components
Pushing v55.0 metadata to test-ovsqkpand1dp@example.com using the v56.0 SOAP API
DEPLOY PROGRESS | ████████████████████████████████████████ | 1/1 Components
Pushing v55.0 metadata to test-ovsqkpand1dp@example.com using the v56.0 SOAP API
DEPLOY PROGRESS | ████████████████████████████████████████ | 4/4 Components

### Actual result
The directories are pushed sequentially with duplicate log messages for each subsequent directory:

Pushing v55.0 metadata to test-ovsqkpand1dp@example.com using the v56.0 SOAP API
DEPLOY PROGRESS | ████████████████████████████████████████ | 2/2 Components
Pushing v55.0 metadata to test-ovsqkpand1dp@example.com using the v56.0 SOAP API
Pushing v55.0 metadata to test-ovsqkpand1dp@example.com using the v56.0 SOAP API
DEPLOY PROGRESS | ████████████████████████████████████████ | 1/1 Components
Pushing v55.0 metadata to test-ovsqkpand1dp@example.com using the v56.0 SOAP API
Pushing v55.0 metadata to test-ovsqkpand1dp@example.com using the v56.0 SOAP API
Pushing v55.0 metadata to test-ovsqkpand1dp@example.com using the v56.0 SOAP API
DEPLOY PROGRESS | ████████████████████████████████████████ | 4/4 Components

### System Information

- Which shell/terminal are you using? (e.g. bash, zsh, powershell 5, powershell 7, cmd.exe, etc.)
```
{
  "cliVersion": "sfdx-cli/7.183.1",
  "architecture": "darwin-x64",
  "nodeVersion": "node-v18.12.1",
  "pluginVersions": [
    "@dxatscale/sfpowerscripts 20.5.2 (user)",
    "@eon-com/eon-sfdx 1.9.4 (user)",
    "@oclif/plugin-autocomplete 1.3.8 (core)",
    "@oclif/plugin-commands 2.2.2 (core)",
    "@oclif/plugin-help 5.1.20 (core)",
    "@oclif/plugin-not-found 2.3.13 (core)",
    "@oclif/plugin-plugins 2.1.8 (core)",
    "@oclif/plugin-search 0.0.6 (core)",
    "@oclif/plugin-update 3.0.9 (core)",
    "@oclif/plugin-version 1.1.4 (core)",
    "@oclif/plugin-warn-if-update-available 2.0.18 (core)",
    "@oclif/plugin-which 2.2.6 (core)",
    "1commerce 0.0.10 (user)",
    "alias 2.1.14 (core)",
    "apex 1.3.0 (core)",
    "auth 2.3.8 (core)",
    "community 2.0.18 (core)",
    "config 1.4.22 (core)",
    "custom-metadata 2.0.5 (core)",
    "data 2.1.19 (core)",
    "generator 2.0.13 (core)",
    "info 2.3.1 (core)",
    "limits 2.2.0 (core)",
    "org 2.2.20 (core)",
    "packaging 1.12.2 (core)",
    "schema 2.2.2 (core)",
    "signups 1.2.12 (core)",
    "source 2.3.12 (core)",
    "telemetry 2.0.5 (core)",
    "templates 55.1.0 (core)",
    "trust 2.2.4 (core)",
    "user 2.1.24 (core)",
    "@salesforce/sfdx-plugin-lwc-test 1.0.1 (core)",
    "salesforce-alm 54.8.5 (core)",
    "sfdmu 4.23.2 (user)",
    "sfpowerkit 6.0.1 (user)"
  ],
  "osVersion": "Darwin 21.6.0",
  "shell": "zsh",
  "rootPath": "/Users/user/.local/share/sfdx/client/7.183.1-3a2f5ba"
}
```
### Additional information
![Screenshot 2023-01-13 at 15 17 33](https://user-images.githubusercontent.com/97173615/212340923-0747ac2f-dc01-444e-9657-d2c931fd8d10.png)

**Comment by WillieRuemmele** [comment]
adding onto this, I found it seems related to `apiversion` 
```
 ➜  sfdx force:source:deploy -p force-app --apiversion=57.0 
(node:71680) Warning: The requested API version (57.0) is higher than the org supports.  Using 56.0.
(Use `node --trace-warnings ...` to show where the warning was created)
Deploying v55.0 metadata to test-y2wsfd5m2no3@example.com using the v56.0 SOAP API
Deploy ID: 0Af8G00000UQfwNSAT
(node:71680) Warning: The requested API version (57.0) is higher than the org supports.  Using 56.0.
(node:71680) Warning: The requested API version (57.0) is higher than the org supports.  Using 56.0.
DEPLOY PROGRESS | ███░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░ | 6/89 Components(node:71680) Warning: The requested API version (57.0) is higher than the org supports.  Using 56.0.
(node:71680) Warning: The requested API version (57.0) is higher than the org supports.  Using 56.0.
(node:71680) Warning: The requested API version (57.0) is higher than the org supports.  Using 56.0.
(node:71680) Warning: The requested API version (57.0) is higher than the org supports.  Using 56.0.
(node:71680) Warning: The requested API version (57.0) is higher than the org supports.  Using 56.0.
DEPLOY PROGRESS | ██████████░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░ | 23/89 Components(node:71680) Warning: The requested API version (57.0) is higher than the org supports.  Using 56.0.
(node:71680) Warning: The requested API version (57.0) is higher than the org supports.  Using 56.0.
(node:71680) Warning: The requested API version (57.0) is higher than the org supports.  Using 56.0.
(node:71680) Warning: The requested API version (57.0) is higher than the org supports.  Using 56.0.
(node:71680) Warning: The requested API version (57.0) is higher than the org supports.  Using 56.0.
DEPLOY PROGRESS | █████████████████████████████░░░░░░░░░░░ | 64/89 Components(node:71680) Warning: The requested API version (57.0) is higher than the org supports.  Using 56.0.
(node:71680) Warning: The requested API version (57.0) is higher than the org supports.  Using 56.0.
DEPLOY PROGRESS | █████████████████████████████████░░░░░░░ | 74/89 Components(node:71680) Warning: The requested API version (57.0) is higher than the org supports.  Using 56.0.
DEPLOY PROGRESS | ██████████████████████████████████░░░░░░ | 75/89 Components(node:71680) Warning: The requested API version (57.0) is higher than the org supports.  Using 56.0.
(node:71680) Warning: The requested API version (57.0) is higher than the org supports.  Using 56.0.
DEPLOY PROGRESS | ███████████████████████████████████░░░░░ | 77/89 Components(node:71680) Warning: The requested API version (57.0) is higher than the org supports.  Using 56.0.
(node:71680) Warning: The requested API version (57.0) is higher than the org supports.  Using 56.0.
(node:71680) Warning: The requested API version (57.0) is higher than the org supports.  Using 56.0.
DEPLOY PROGRESS | ██████████████████████████████████████░░ | 84/89 Components(node:71680) Warning: The requested API version (57.0) is higher than the org supports.  Using 56.0.
DEPLOY PROGRESS | ███████████████████████████████████████░ | 86/89 Components(node:71680) Warning: The requested API version (57.0) is higher than the org supports.  Using 56.0.
(node:71680) Warning: The requested API version (57.0) is higher than the org supports.  Using 56.0.
(node:71680) Warning: The requested API version (57.0) is higher than the org supports.  Using 56.0.
DEPLOY PROGRESS | ████████████████████████████████████████ | 89/89 Components(node:71680) Warning: The requested API version (57.0) is higher than the org supports.  Using 56.0.
(node:71680) Warning: The requested API version (57.0) is higher than the org supports.  Using 56.0.
```

from a single package project (dreamhouse)

**Comment by shetzel** [comment]
UPDATE: This is fixed in the current CLI release candidate, v7.186.0 which will be promoted in a week.

**Comment by petter-eikeland** [comment]
Tested with the newest release candidate. Works like expected! Thank you
